src/components/TeamCard.js

```diff
@@ -1,26 +1,29 @@
-import { useState } from 'react';
-import { Card, CardMedia, CardBody, CardHeader, Title } from '@trussworks/react-uswds'
+import { Card, CardMedia, CardBody, CardHeader } from '@trussworks/react-uswds'
 import '@trussworks/react-uswds/lib/index.css'
 
+import { useTranslation } from "react-i18next";
+import "../translations/i18n";
+
+function TeamCard({name, position}) {
+  const { t } = useTranslation();
 
-function TeamCard({name, position, img_source}) {
   return (
     <>
       <Card
         gridLayout={{ tablet: { col: 4 } }}
         containerProps={{ className: 'border-none' }}>
         <CardHeader>
-          <h3>{name}</h3>
+          <h3>{t(name)}</h3>
         </CardHeader>
         <CardMedia>
           <img
-            src={img_source}
-            alt={`${name}'s headshot`}
+            src={`/assets/${name}.jpg`}
+            alt={t("headshot", { name: name })}
```

Comment:
I think syntax is interesting. It's cool to see how you used this and an array in your constants file instead of an array in the translation file!